sdk/trace/src/main/java/io/opentelemetry/sdk/trace/export/SimpleSpanProcessor.java

```diff
@@ -48,19 +49,30 @@
   private final boolean sampled;
   private final AtomicBoolean isShutdown = new AtomicBoolean(false);
 
+  /**
+   * Returns a new {@link SimpleSpanProcessor} which exports spans to the {@link SpanExporter}
+   * synchronously.
+   */
+  public static SpanProcessor create(SpanExporter exporter) {
```

Comment:
I have no opinion on this point, but often we hid such classes in the past and instead used a static method in the interface. So what about making this class deprecated and later package-private and instead adding a `SpanProcessor.simple(Exporter)` method? (or call it `wrap`)

Reply:
I tend to prefer the pattern but I don't know if we want the `export` package to leak into the top level. It feels a bit weird to me though maybe it's not such a big deal.

A different option with better grouping would be for `SpanExporter` to expose the factory. But also weird. Default methods `asSimpleSpanProcessor` etc seems better.

Another idea in my head is to make this private completely and only have `SdkTracerProvider.addSpanExporter(exporter, BatchSettings.noBatching())`.

Not sure if it works but is also a reason I wanted to simplify this builder away :)

Reply:
I think we should come to a (potentially) final decision here before removing/deprecating any public stuff. The builder is not that big a burden to maintain in the meantime. If we decide to leave the SimpleSpanProcessor for the foreseeable future then this PR is fine, otherwise I'd prefer we leave the builder as-is for now.

Reply:
I think `Exporter.asSimpleSpanProcessor` would be fine. Otherwise, if we somehow want to keep SpanProcessor/Exporter statically decoupled (so far the interfaces don't know about each other), we could add an utility class like `ExportingSpanProcessor` with a static method `simple` (and `batching`).